### PR TITLE
[new] Add unsplash-js

### PR DESCRIPTION
**Pull request by jsejcksn**
Pull request for issue: I created this PR directly instead of creating a request issue
Related issue(s): 

## Profile of the lib

* Git repository (required): https://github.com/unsplash/unsplash-js
* Official website (optional, not the repository): https://unsplash.com/developers
* NPM package url (optional): https://www.npmjs.com/package/unsplash-js
* License and its reference: [MIT](https://github.com/unsplash/unsplash-js/blob/master/LICENSE)
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 22
  * Count of stars: 847
  * Count of forks: 65
* NPM download stats (optional):
  * Downloads in the last day:
  * Downloads in the last week: 6721
  * Downloads in the last month:

## Essential checklist

* [ ] I'm the author of this library
  * [x] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [x] The lib has notable popularity
  * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [x] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [ ] Has valid tags for each versions (for git auto-update)
* [ ] Auto-update setup
* [ ] Auto-update target/source is valid.
* [ ] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [x] Pull request is sent from a non-master branch with a meaningful name.
* [x] Separate unrelated changes into different commits.
* [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by jsejcksn** [comment]
I followed the [contributing guidelines](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#b-pull-request), but I'm still unsure about the auto-update portion, as I am not the author of the library.

Following the guidelines section [adding a new library by a single package.json](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#e-adding-a-new-library-by-a-single-packagejson), the only modifications I made to the source [`package.json`](https://github.com/cdnjs/cdnjs/blob/ceda719b548369cfade4b00de86b5c71e853f967/ajax/libs/unsplash-js/package.json#L3) file were:
  - Removing the `version` property
  - Adding the `filename` property

Can someone give me some guidance/feedback on which changes need to be made, if any? Thanks.

**Comment by MattIPv4** [comment]
Hi @jsejcksn 

As stated in the CI error reported the following things are good steps to get this PR ready:

`must have a valid auto-update config` - Please see the CONTRIBUTING-WIP.md file in our repo or existing package.json files from other libraries on here for how auto-update works.

`we don't need main, scripts, bugs, devDependencies, dependencies keys in package.json` - Please again look at existing package.json files for libraries here to ascertain which keys we need :)

`Homepage doesn't need to be set if it's the same as repository` - you can use https://unsplash.com/developers as the homepage instead?

File: ajax/libs/unsplash-js/package.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
+  "name": "unsplash-js",
+  "filename": "unsplash.min.js",
+  "description": "A Universal JavaScript wrapper for the Unsplash API",
+  "main": "lib/unsplash.js",
```
Comment (MattIPv4) [inline ajax/libs/unsplash-js/package.json]:
```suggestion
```
This attr is not needed.

File: ajax/libs/unsplash-js/package.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    "test:browser": "karma start",
+    "test:ci": "npm run lint && npm run flow && npm run test:browser && npm run test:coverage",
+    "test:coverage": "babel-node ./node_modules/.bin/isparta cover _mocha -- --require test/setup",
+    "test:node": "mocha --compilers js:babel-core/register --require test/setup --recursive",
+    "test:watch": "npm test -- --watch",
+    "prepublish": "npm run build",
+    "publish:major": "npm run build && npm version major",
+    "publish:minor": "npm run build && npm version minor",
+    "publish:patch": "npm run build && npm version patch"
+  },
```
Comment (MattIPv4) [inline ajax/libs/unsplash-js/package.json]: This attr is not needed.

File: ajax/libs/unsplash-js/package.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    "karma-firefox-launcher": "0.1.7",
+    "karma-mocha": "0.2.2",
+    "karma-safari-launcher": "0.1.1",
+    "karma-sourcemap-loader": "0.3.7",
+    "karma-webpack": "1.7.0",
+    "mocha": "2.3.3",
+    "mockery": "1.4.0",
+    "node-fetch": "1.5.0",
+    "webpack": "1.12.14"
+  },
```
Comment (MattIPv4) [inline ajax/libs/unsplash-js/package.json]: This attr is not needed.

File: ajax/libs/unsplash-js/package.json [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  "dependencies": {
+    "form-urlencoded": "1.2.0",
+    "lodash.get": "4.4.2",
+    "querystring": "0.2.0",
+    "url-parse": "1.2.0"
+  }
```
Comment (MattIPv4) [inline ajax/libs/unsplash-js/package.json]: This attr is not needed.

**Comment by jsejcksn** [comment]
@MattIPv4 Thanks for the review comments.

It's been a long time since I've looked at this PR. In retrospect, I think I should have created an Issue to request the library instead of opening a PR: I'm not familiar with some of the `git` aspects described in [CONTRIBUTING](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#b-pull-request) so I don't have any intentions of going further with addressing the CI build issues—I just wanted to improve the possibility of its inclusion. If there's a way to reassign the PR to someone else, let me know. Otherwise, I can just close it.

**Pull request closed without merging**